beetsplug/deezer.py

```diff
@@ -112,8 +111,8 @@ def album_for_id(self, album_id):
             day = None
         else:
             raise ui.UserError(
-                "Invalid `release_date` returned "
-                "by {} API: '{}'".format(self.data_source, release_date)
+                f"Invalid `release_date` returned by {self.data_source} API: "
+                f"{release_date!r}"
```

Comment:
The `!r` component is new, is it intentional?

Reply:
Yep - you can see this replaced the single quotes in the message as `__repr__` for strings adds them.